test/old-start-configs/v1.0.0/test-end-to-end.sh

```diff
@@ -124,17 +124,13 @@ function cleanup()
 	echo
 
 	if [[ -z "${SKIP_TEARDOWN-}" ]]; then
-		echo "[INFO] Deleting test constructs"
-		oc delete -n test all --all
-		oc delete -n docker all --all
-		oc delete -n custom all --all
-		oc delete -n cache all --all
-		oc delete -n default all --all
-
 		echo "[INFO] Tearing down test"
 		pids="$(jobs -pr)"
 		echo "[INFO] Children: ${pids}"
-		sudo kill ${pids}
+		for i in ${pids[@]}; do
```

Comment:
> Shouldn't we use ${pids[@]-} instead of ${pids[@]} here too?

That one always seems to have a value and this is breaking our extended test runs.  Do you mind if I merge this on green and fix that up next time I'm in the file?